src/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+add_library(pplib_utils STATIC
+    util/utilcrypt.c
+    util/utiliof.c
+    util/utillog.c
+    util/utilmd5.c
+    util/utilmem.c
+    util/utilnumber.c
+    util/utilsha.c
+)
+
+target_include_directories(pplib_utils PUBLIC
```

Comment:
(Just for clarification, not request for changing anything: I'm not absolutely sure if pplib in fact consists of three independent libs. This is just what I saw in the makefile and tried to be somewhat consistent.)

Reply:
pplib depends only on zlib, nothing else. It uses a code from src/util directory, but src/util is a toolbox, not build as a separate lib (a set of tools independent from PDF, eg. md5/sha2, compresion filters and such). But of course can be seen as such by a build system.